From foo@baz Thu Jun  4 11:29:54 PDT 2009
Date: Thu, 04 Jun 2009 11:29:54 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: rtl8192su: fix build warnings

This fixes some build warnings in the rtl8192su driver.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c
+++ b/drivers/staging/rtl8192su/ieee80211/ieee80211_wx.c
@@ -731,7 +731,7 @@ int ieee80211_wx_set_encode_ext(struct i
 
 	 ops = ieee80211_get_crypto_ops(alg);
         if (ops == NULL) {
-                request_module(module);
+                request_module("%s", module);
                 ops = ieee80211_get_crypto_ops(alg);
         }
         if (ops == NULL) {
